Source/EasyNetQ/ConnectionConfiguration.cs

```diff
@@ -90,21 +92,29 @@ private void SetDefaultClientProperties(IDictionary<string, object> clientProper
             var hostname = Environment.MachineName;
             var product = Product ?? applicationName;
             var platform = Platform ?? hostname;
+            var name = Name ?? applicationName;
```

Comment:
Well I made it intentionaly non empty in order to easier to distinguish multiple clients in rabbitmq web management console. So no changes would be needed to see which app is connected.
But of course if you think it is better to leave it empty I dont mind.
Btw, any plans to upgrade RabbitMq client library as some version have a bug that connection_name is not sent to server.